site/content/en/docs/tasks/operations/standalone-deployment-mode.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Standalone Deployment Mode"
+---
+
+{{% alert title="Notice" color="warning" %}}
+
+Standalone mode is an experimental feature, please **DO NOT** use it in production.
+
+{{% /alert %}}
+
+Envoy Gateway also supports running in standalone mode. In this mode, Envoy Gateway
+does not need to rely on Kubernetes and can be deployed directly on bare metal or virtual machines.
+In the standalone deployment mode, the functions and features supported by Envoy Gateway are 
+basically the same as those in the Kubernetes deployment mode.
```

Comment:
my suggestion would be to rm L13-14 or specifically specify which features will not work in standalone mode

Reply:
removed